src/pipeline_configuration_spec.py

```diff
@@ -22,11 +22,35 @@
                                                          CodingConfiguration, GoogleDriveUploadConfiguration,
                                                          MembershipGroupConfiguration, AnalysisConfiguration)
 
+
 def load_code_scheme(fname):
     with open(f"code_schemes/{fname}.json") as f:
         return CodeScheme.from_firebase_map(json.load(f))
 
 
+def make_rqa_coda_dataset_configs(dataset_name_prefix, coda_dataset_id_prefix, code_scheme_prefix, number_of_datasets):
```

Comment:
This is fine in the rvi repo, but I'm not sure we'd want in the main test-pipeline repo because it's quite a brittle piece of configuration support.

If this is copied from create_imaqal_pool do you want to remove it from there and have that reference this implementation?